sympy/series/limits.py

```diff
@@ -136,6 +136,11 @@ def doit(self, **hints):
         if not e.has(z):
             return e
 
+        # limit can work with binomial by converting to factorial first
+        if (e.has(binomial) and not (hints.get('sequence', True)
```

Comment:
Just ``if e.has(binomial)`` would be better. I would prefer ``limit_seq`` to be a last resort.